### ISSUE TITLE
Update spec to refer to approved MIME type (once approved)

### ISSUE THREAD

**Issue by michaelcfanning** [oasis-tcs/sarif-spec#182]
The proposal is for application/sarif+json

**Comment by robincover** [comment]
ACK receipt of email requesting coordination on registration process w/ IANA

**Comment by michaelcfanning** [comment]
See #118. Do we need a new MIME type for SARIF 'fragments'?

**Comment by lgolding** [comment]
Yes. I propose `application/sarif-fragment+json`

**Comment by lgolding** [comment]
- Register `application/sarif+json` with IANA as the MIME type for SARIF log files, and update the spec to mention it.
- Register the `sarif` URI scheme with IANA, and mention the registration in the spec section the describes the scheme.

Closing #342 as duplicate of this issue.

@michaelcfanning @kupsch

**Comment by michaelcfanning** [comment]
@lgolding nudging this one. where are we at?

**Comment by lgolding** [comment]
@michaelcfanning I just wrote to @chet-ensign asking how to proceed, cc-d you, @dmk42, and Luke.

**Comment by lgolding** [comment]
The media type `application/sarif+json` is now registered.

@michaelcfanning Do we still want to go on to register the `sarif` URI scheme? If so I'll close this issue and open a separate one for that; otherwise I'll just close this issue.

**Comment by michaelcfanning** [comment]
Yes, we should complete the work initially proposed in this issue. Let's just leave it open until it is complete.

**Comment by lgolding** [comment]
@michaelcfanning As we discussed, the proposed media type `application/sarif-fragment+json` proposed above should be `application/sarif-external-properties+json`, matching the filename extension `.sarif-external-properties` for external property files.

**Comment by bernhardreiter** [comment]
> we should complete the registration. Let's just leave this issue open until it is complete.

The registration for 
 * https://www.iana.org/assignments/media-types/application/sarif-external-properties+json
 * https://www.iana.org/assignments/media-types/application/sarif+json
 * https://www.iana.org/assignments/uri-schemes/prov/sarif

appears to be done (since 2021), so this issue appears to be resolved.
